contracts/starknet/SpaceAccount.cairo

```diff
@@ -277,6 +281,14 @@ func update_proposal_threshold{
     return ();
 }
 
+@external
+func update_metadata_uri{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr: felt}(
```

Comment:
I think it should be called `set_metadata_uri`, following OZ standard for NFT with tokenURI and setTokenURI

Reply:
I think Orlando used `update` to go along with with other `update_*` functions we have. Do you think we should rename all `update_*` to `set_*` ?

Reply:
the rest of the space parameters have `update` rather than `set` functions. why do we need to keep to that standard? doesnt seem relevant here

Reply:
If we use "update" everywhere we can use it here too, but I recommend move to set everywhere at some point. It's just much shorter, more common.